### DISCUSSION TITLE
Record History [Jun 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2486] [Discord Threads]
> 🙋‍♂️ **esteve**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
####   
![image0](https://cdn.discordapp.com/attachments/987264841825808444/987291224274767892/unknown.png)  
#  
> 🙋‍♂️ **wkw**  
#### 1. Expand the record   
#### 2. next to cancel, there is a button called toggle comments  
![image0](https://cdn.discordapp.com/attachments/987264841825808444/987291501740556318/unknown.png)  
![image1](https://cdn.discordapp.com/attachments/987264841825808444/987291502042550292/unknown.png)  
#  
> 🙋‍♂️ **esteve**  
#### Thank you so much  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/987264841825808444)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
